Blockzilla/BrowserViewController.swift

```diff
@@ -187,7 +187,7 @@ class BrowserViewController: UIViewController {
         createURLBar()
         
         // Listen for request desktop site notifications
-        let nc = NotificationCenter.default.addObserver(forName: Notification.Name(rawValue: UIConstants.strings.requestDesktopNotification), object: nil, queue: nil)  { _ in
+        let _ = NotificationCenter.default.addObserver(forName: Notification.Name(rawValue: UIConstants.strings.requestDesktopNotification), object: nil, queue: nil)  { _ in
```

Comment:
Can you remove the `let _ =` all together? Or is a linter getting angry?

Reply:
Good call, fixed this.